.github/workflows/openapi-docs-pr.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         working-directory: schemas
     steps:
       - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
-      - uses: actions/setup-node@1f8c6b94b26d0feae1e387ca63ccbdc44d27b561
+      - uses: actions/setup-node@9ced9a43a244f3ac94f13bfd896db8c8f30da67a
         with:
           node-version: 16
```

Comment:
did we want to remove these, as the default is now 16?

Reply:
done